docs/contributor/resources/03-moduletemplate.md

```diff
@@ -75,6 +75,32 @@ spec:
 - documentation: The link to the documentation of the module.
 - icons: A list of icons of the module, each with a name and link.
 
+### **.spec.manager**
+
+The `manager` field provides information for identifying a module's resource that can be used as indicator for the installation readiness of the module. Typically, this is the manager Deployment of the module. In exceptional cases, it may also be another resource. If the resource is not namespaced, such as the module CR's `CRD`, the namespace can be optional.
+
+For example, if the module has a manager `Deployment` in the `kyma-system` namespace:
+
+```yaml
+spec:
+  manager:
+    group: apps
+    version: v1
+    kind: Deployment
+    namespace: kyma-system
+    name: [module manager name]
+```
+
+For a module without a manager, such as [cloud manager](https://github.com/kyma-project/cloud-manager):
```

Comment:
```suggestion
In this example, the module doesn't have a manager and in the **manager** field you specifie the module's `CustomResourceDefinition`. For example, of the [Cloud Manager](https://github.com/kyma-project/cloud-manager) module:
```
I'm just not sure if it means that there is no manager or if the manager is the CRD?